src/ape/api/providers.py

```diff
@@ -23,6 +23,7 @@
 from ape.api.config import PluginConfig
 from ape.api.networks import LOCAL_NETWORK_NAME, NetworkAPI
 from ape.api.transactions import ReceiptAPI, TransactionAPI
+from ape.api.query import BlockTransactionQuery
```

Comment:
Oh, wait this is causing the circular import? Then yeah, I would just import it inside of `transactions` below. That should resolve it